src/base_model.jl

```diff
@@ -280,28 +280,27 @@ function build_base_model!(ECModel::AbstractEC, optimizer; use_notations=false)
         P_conv_P_us[u, c, t] - P_conv_N_us[u, c, t]
     )
 
-    # Total adjustable load dispatch for each appliance: positive when charging the vehicle
-    @expression(model_user, P_adj_us[u=user_set, e=asset_names(users_data[u], LOAD_ADJ), t=time_set],
-        P_adj_P_us[u, e, t] - P_adj_N_us[u, e, t]
-    )
-
-    # Total energy load by user and time step for adjustable load: positive when charging the vehicle
-    @expression(model_user, P_adj_tot_us[u=user_set, t=time_set],
-        sum(P_adj_us[u,e,t] for e in asset_names(users_data[u], LOAD_ADJ))
-    )
-    
-    # Fixed power for single fixed appliance by user
-    @expression(model_user, P_fix_us[u=user_set, f=asset_names(users_data[u], LOAD), t=time_set],
-        profile_component(users_data[u], f, "load")[t])
-
-    # Total energy load by user and time step for fixed load
-    @expression(model_user, P_fix_tot_us[u=user_set, t=time_set],
-        sum(P_fix_us[u,e,t] for e in asset_names(users_data[u], LOAD))
+    # Overestimation of the power exchanged by each POD when selling to the external market by each user
+    # TODO modify this expression to include adj_load
+    @expression(model_user, P_P_us_overestimate[u in user_set, t in time_set],
+        max(0,
+            sum(Float64[field_component(users_data[u], c, "max_capacity") 
+                for c in asset_names(users_data[u], CONV)]) # Maximum capacity of the converters
+            + sum(Float64[field_component(users_data[u], r, "max_capacity")*profile_component(users_data[u], r, "ren_pu")[t] 
+                for r = asset_names(users_data[u], REN)]) # Maximum dispatch of renewable assets
+            + sum(Float64[field_component(users_data[u], g, "max_capacity")*field_component(users_data[u], g, "max_technical")
+                for g = asset_names(users_data[u], THER)]) #Maximum dispatch of the fuel-fired generators
+            - sum(P_L_tot_us[u, t])  # Minimum demand
+        ) * TOL_BOUNDS
     )
 
-    # Total energy load by user and time step
-    @expression(model_user, P_L_tot_us[u=user_set, t=time_set],
-        P_adj_tot_us[u,t] + P_fix_tot_us[u,t]
+    # Overestimation of the power exchanged by each POD when buying from the external market by each user
+    @expression(model_user, P_N_us_overestimate[u in user_set, t in time_set],
+        max(0,
+            sum(P_L_tot_us[u, t]) # Maximum demand

```

Comment:
On this, P_L_tot_us[u, t] is not a constant number but a real expression.
The scope of overestimate is to have a basic numerical overestimate of dispatch variables.

For the overestimate N, we can estimate it by summing the profile load(as before) and the maximum charging profile of the adj load.
For the overestimate P, the adj contributes with the maximum discharging profile in general [in the italian context, that may not be the case, but we can generalize it to start]